frontend/src/components/AccountReceiveTable.vue

```diff
@@ -594,19 +594,12 @@ function useReceivedFundsTable(announcements: UserAnnouncement[], spendingKeyPai
           relayTransactionHash: string;
         };
 
-        if (chainId === 137) {
-          // No relayer support on this network, so this is a regular transaction hash
-          console.log(`${vm.$i18n.t('AccountReceiveTable.relayed-with-tx-hash').toString()} ${relayTransactionHash}`);
-          const receipt = await provider.value.waitForTransaction(relayTransactionHash);
-          console.log(vm.$i18n.t('AccountReceiveTable.withdraw-successful-receipt'), receipt);
-        } else {
-          // Received a relayer transaction hash, wait for withdraw transaction to be mined
-          console.log(
-            `${vm.$i18n.t('AccountReceiveTable.relayed-with-relayer-id').toString()} ${relayTransactionHash}`
-          );
-          const { receipt } = (await relayer.value?.waitForId(relayTransactionHash)) as ConfirmedRelayerStatusResponse;
-          console.log(vm.$i18n.t('AccountReceiveTable.withdraw-successful-receipt'), receipt);
-        }
+        // This is a regular transaction hash, though it's possible OZ Defender will replace it to ensure it gets
+        // included quickly, in which case the frontend would not automatically reflect when the relay was successful.
+        // Because we relay with the "fast" setting, this is unlikely to be the case.
+        console.log(`${vm.$i18n.t('AccountReceiveTable.relayed-with-tx-hash').toString()} ${relayTransactionHash}`);
+        const receipt = await provider.value.waitForTransaction(relayTransactionHash);
+        console.log(vm.$i18n.t('AccountReceiveTable.withdraw-successful-receipt'), receipt);
```

Comment:
I know you didn't change this, but we should gradually migrate from `console.log()` to our logger class. Any objections on replacing the console.logs with `window.logger.info()`? (debug and warn are other possible options, but info seems right here)

Reply:
Sure! Done and amended into the commit.

Reply:
Appreciate it!